index.js

```diff
@@ -0,0 +1,149 @@
+const compareLangKeys = () => {
+  // lib is needed to show deuplicated keys
```

Comment:
Consider removing this kind of comments as they carry little information that is really needed. Others will follow the usages to see what this is used for.

```suggestion
  // lib is needed to show duplicated keys
```